src/templates/pycontw-2020/_includes/portal.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+
+<div class="card">
+	<a class="button-round" href="{% url 'events_schedule' %}" class="card-link">{% trans 'Schedule' %}</a>
+</div>
+
+<div class="card">
+	<a class="button-round" href="https://hackmd.io/@PyConTW/2019" class="card-link">{% trans 'Collaborative Notes' %}</a>
```

Comment:
2019 --> 2020

Reply:
I'll edit it ,by the way ,this page is not found now ~